integrationTests/vm/delegation/delegationScenarios_test.go

```diff
@@ -31,6 +31,38 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
+func TestDelegationSystemNodesOperationsTest(t *testing.T) {
+	tpn := integrationTests.NewTestProcessorNode(1, core.MetachainShardId, 0)
+	tpn.InitDelegationManager()
+	maxDelegationCap := big.NewInt(5000)
+	serviceFee := big.NewInt(1000)
+	value := big.NewInt(1000)
+
+	tpn.BlockchainHook.SetCurrentHeader(&block.MetaBlock{Nonce: 1})
+
+	// create new delegation contract
+	delegationScAddress := deployNewSc(t, tpn, maxDelegationCap, serviceFee, value, tpn.OwnAccount.Address)
+
+	scrsHandler, _ := tpn.InterimProcContainer.Get(block.SmartContractResultBlock)
+	scrs := scrsHandler.GetAllCurrentFinishedTxs()
+
+	numExpectedScrsFound := 0
+	// we expect 2 scrs:
```

Comment:
is this valid?

Reply:
yes, changed to `we expect the following 2 scrs`, as there can be an additional scr (for refunded gas), but it is not in the scope of this test